SocialFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,7 @@
 !
 ! Extended CSS rules
 !
+winaero.com#?#.share:not(.share:contains(Support us))
```

Comment:
Better without negation(in general safer)
`winaero.com#?#.share:has(> p:contains(Share this post))`

Reply:
In different perspective, both are right.